app/network/entity/entity.form.component.js

```diff
@@ -229,5 +233,72 @@
     function storeGroups(groups) {
       _.setContents(entityForm.groups.items, groups);
     }
+
+    function incrementBulk() {
+      var is_bulk = entityForm.v4.is_bulk;
+      var increment_vlan = entityForm.v4.increment_vlan;
+      var ip_increment = entityForm.input.v4.increment_amount;
+
+      if (is_bulk) {
+        if (ip_increment) {
+          var wholeIP = entityForm.input.v4.address;
+          var ip = wholeIP.split('.');
+          var last = parseInt(ip.pop());
```

Comment:
you can remove the next line and just use:

```suggestion
          var last = parseInt(ip.pop()) + ip_increment;
```